package cs.vapo.bringit.core.exceptions;

public class InternalServerException extends RuntimeException {

    public InternalServerException(final Throwable cause) {
        super(cause);
    }
}
